mi-scheduler/overlays/stage/test/buildConfig.yaml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
what is overlays/stage/test? :D We don't test in stage :)

Reply:
and we dont want a buildconfig in stage, just in test...

Reply:
if you are adding to the stage environment, please add a corresponding PR for https://github.com/AICoE/aicoe-cd/tree/master/manifests/overlays/prod/applications/thoth-station/stage

Reply:
maaan, i somehow didn't notice that i had invalid structure of stage/test :facepalm: